soperator/modules/slurm/variables.tf

```diff
@@ -248,6 +248,12 @@ variable "accounting_enabled" {
   default     = false
 }
 
+variable "use_protected_secret" {
+  description = "If true, protected secret MariaDB will be deleted after the MariaDB CR is deleted."
```

Comment:
I don't understand why is it called `protected secret` if it is being deleted with CR